app/ui/src/main/java/de/mm20/launcher2/ui/launcher/search/location/LocationItem.kt

```diff
@@ -372,24 +374,10 @@ fun LocationItem(
                     }
 
                     val openingSchedule = location.openingSchedule
-                    // remember?
-                    val departures = location.departures.or { debugDepartures() }
-                        ?.sortedBy { it.time }
-                        ?.let {
-                            val morningDepartures =
-                                it.filter { LocalTime.MIDNIGHT <= it.time && it.time <= LocalTime.NOON }
-                            val afternoonDepartures =
-                                it.filter { LocalTime.NOON < it.time && it.time < LocalTime.MIDNIGHT }
-
-                            val now = LocalTime.now()
-                            val isAfternoon = LocalTime.NOON < now && now < LocalTime.MIDNIGHT
-
-                            if (isAfternoon) {
-                                afternoonDepartures + morningDepartures
-                            } else {
-                                morningDepartures + afternoonDepartures
-                            }
-                        }
+                    val departures = remember(location.departures) {
+                        location.departures
+                            ?.sortedBy { it.time }
+                    }
```

Comment:
Does switching to `ZonedDateTime` account for departures past midnight? E.g.: it is 11:45 PM and some departures returned by a provider are past midnight